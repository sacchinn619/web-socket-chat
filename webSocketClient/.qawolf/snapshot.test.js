const qawolf = require("qawolf");

let browser;
let page;

beforeAll(async () => {
  browser = await qawolf.launch();
  const context = await browser.newContext();
  await qawolf.register(context);
  page = await context.newPage();
});

afterAll(async () => {
  await qawolf.stopVideos();
  await browser.close();
});

test("snapshot", async () => {
  await page.goto("http://localhost:1992/");
  await page.click("#search");
  await page.fill("#search", "Bob");
  await page.press("#search", "Enter");
  await page.click(".ant-input");
  await page.fill(".ant-input", "Hey!");
  await page.press(".ant-input", "Enter");
  await page.fill(".ant-input", "Hi there!");
  await page.press(".ant-input", "Enter");
  const hits = await page.$('#messages');
  expect(await hits.evaluate((node) => node.outerHTML)).toMatchSnapshot();
});