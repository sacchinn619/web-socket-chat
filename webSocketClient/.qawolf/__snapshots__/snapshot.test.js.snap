// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `"<div id=\\"messages\\" style=\\"display: flex; flex-direction: column; padding-bottom: 50px;\\"><div class=\\"ant-card ant-card-bordered\\" style=\\"width: 300px; margin: 16px 4px 0px; align-self: flex-end;\\"><div class=\\"ant-card-body\\"><div class=\\"ant-card-meta\\"><div class=\\"ant-card-meta-avatar\\"><span class=\\"ant-avatar ant-avatar-circle\\" style=\\"color: rgb(245, 106, 0); background-color: rgb(253, 227, 207);\\"><span class=\\"ant-avatar-string\\" style=\\"transform: scale(1) translateX(-50%);\\">B</span></span></div><div class=\\"ant-card-meta-detail\\"><div class=\\"ant-card-meta-title\\">Bob</div><div class=\\"ant-card-meta-description\\">Hey!</div></div></div></div></div><div class=\\"ant-card ant-card-bordered\\" style=\\"width: 300px; margin: 16px 4px 0px; align-self: flex-end;\\"><div class=\\"ant-card-body\\"><div class=\\"ant-card-meta\\"><div class=\\"ant-card-meta-avatar\\"><span class=\\"ant-avatar ant-avatar-circle\\" style=\\"color: rgb(245, 106, 0); background-color: rgb(253, 227, 207);\\"><span class=\\"ant-avatar-string\\" style=\\"transform: scale(1) translateX(-50%);\\">B</span></span></div><div class=\\"ant-card-meta-detail\\"><div class=\\"ant-card-meta-title\\">Bob</div><div class=\\"ant-card-meta-description\\">Hi there!</div></div></div></div></div></div>"`;
